src/app/containers/ATIAnalytics/index.jsx

```diff
@@ -18,7 +19,8 @@ const ATIAnalytics = ({ data }) => {
     article: buildArticleATIUrl,
     frontPage: buildFrontPageATIUrl,
     media: buildRadioATIUrl,
-    MAP: buildCpsAssetPageATIUrl,
+    MAP: buildMAPCpsAssetPageATIUrl,
+    PGL: buildPGLCpsAssetPageATIUrl,
```

Comment:
From my previous suggestion, I was expecting something like: 
```
MAP: buildCpsAssetPageATIUrl({ contentType: 'article-media-asset' }),
PGL: buildCpsAssetPageATIUrl({ contentType: 'article-photo-gallery' }),
```
Am I right in saying that is the only difference between the two asset types?
Just seems like there isn't a need to make two `buildParams` implementations if that is the only difference?